packages/calcite-components/src/components/color-picker/color-picker.tsx

```diff
@@ -1400,14 +1401,29 @@ export class ColorPicker
     context.arc(x, y, radius, startAngle, endAngle);
     context.fillStyle = "#fff";
     context.fill();
+
     context.strokeStyle = "rgba(0,0,0,0.3)";
     context.lineWidth = outlineWidth;
     context.stroke();
 
+    const pattern = context.createPattern(this.getCheckeredBackgroundPattern(), "repeat");
+
+    if (color.alpha() < 1 && applyAlpha) {
```

Comment:
Nitpick: this isn't applicable in this instance since `alpha()` isn't expensive, but in some cases it's beneficial to test flags before functions to avoid the method call.